src/main/resources/db/migration/courtcase/U1117__add_immutable_offence_table.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN;
+-- NOTE: Undo migrations are only supported in the paid version of Flyway so we'll have to convert this to a versioned
+-- migration if we want it applied. It was created as a fallback so we can deploy quickly should we need it.
+CREATE TABLE immutable_court_case_bk AS TABLE court_case;
+CREATE TABLE immutable_offence_bk AS TABLE offence;
+
+DROP TABLE offence;
+DROP TABLE court_case;
+ALTER TABLE offence_bk RENAME TO offence;
+ALTER TABLE court_case_bk RENAME TO court_case;
```

Comment:
Can we remove the "VERSION" column from court_case ? It looks like it's gone from the offence table. It's obviously meaningless now